components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -299,13 +505,7 @@ fn migrate_logins(
     Ok(metrics)
 }
 
-// This was copied from import_multiple in db.rs with a focus on LocalLogin
-fn migrate_local_login(
-    db: &LoginDb,
-    conn: &Connection,
-    local_login: &LocalLogin,
-    encdec: &EncryptorDecryptor,
-) -> Result<()> {
+fn migrate_local_login(conn: &Connection, local_login: &LocalLogin) -> Result<()> {
```

Comment:
a bit of a nit, but I don't think `migrate` is great here as part of the name - maybe replace `migrate` with `insert` here?